### PR TITLE
Create SentinelLinearSearch.cpp

### PR DESCRIPTION
**Pull request by visakhsuku**
**Fixes issue:** #[Mention the issue number it fixes or add the details of the changes if it doesn't has a specific issue.]

**Changes:**
[Add here what changes were made in this pull request.]

### CODE REVIEW

File: code/search/linear_search/SeninelLinearSearch.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/*
+ * Part of Cosmos by OpenGenus Foundation
+*/
+
+/*
+ * Input: The number of elements in an array, The element to be searched, An integer array.
+ * Output: if found returns "found" else "not found", using the sentinel linear search algorithm.
+*/
+
+#include <bits/stdc++.h>
```
Comment (Vitorvgc) [inline code/search/linear_search/SeninelLinearSearch.cpp]: This header shouldn't be used due to portability issues.

**Comment by visakhsuku** [comment]
made the changes you suggested

**Comment by visakhsuku** [comment]
Will make changes according to the style soon :+1:

**Comment by visakhsuku** [comment]
Updated according to style guide, please review

File: code/search/linear_search/SentinelLinearSearch.cpp [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ * Author: Visakh S
+ * Github: visakhsuku
+ * Input: The number of elements in an array, The element to be searched, An integer array.
+ * Output: if found returns "found" else "not found", using the sentinel linear search algorithm.
+*/
+
+#include <iostream>
+#include <vector>
+using namespace std;
+void SentinalLinearSearch(std::vector<int> v,int n,int x)
```
Comment (Vitorvgc) [inline code/search/linear_search/SentinelLinearSearch.cpp]: It should be named `sentinelLinearSearch`.

File: code/search/linear_search/SentinelLinearSearch.cpp [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+{
+	int last,i=0;
+	last=v[n-1];
+	v[n-1]=x;
+	while(v[i]!=x)
+	{
+		i++;
+	}
```
Comment (Vitorvgc) [inline code/search/linear_search/SentinelLinearSearch.cpp]: These braces can be omitted.

**Comment by visakhsuku** [comment]
Made changes as requested, please review

**Comment by visakhsuku** [comment]
not merged??

**Comment by visakhsuku** [comment]
any corrections??

**Comment by Vitorvgc** [comment]
In my opinion it is OK to merge, but I don't have write access.
Please wait until @AdiChat evaluate it.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**